src/run.jl

```diff
@@ -43,11 +62,107 @@ julia> sol.t[end]
 0.1
 ```
 """
-function trixi_include(mod::Module, parameters_file::AbstractString; kwargs...)
-  Base.include(ex -> replace_assignments(ex; kwargs...), mod, parameters_file)
+function trixi_include(mod::Module, elixir::AbstractString; kwargs...)
+  Base.include(ex -> replace_assignments(ex; kwargs...), mod, elixir)
 end
 
-trixi_include(parameters_file::AbstractString; kwargs...) = trixi_include(Main, parameters_file; kwargs...)
+trixi_include(elixir::AbstractString; kwargs...) = trixi_include(Main, elixir; kwargs...)
+
+
+"""
+    convergence_test(elixir::AbstractString, iterations; kwargs...)
+
+Run multiple Trixi simulations with the parameters given in `elixir` and compute
+the experimental order of convergence (EOC) in the ``L^2`` and ``L^\\infty`` norm.
+The number of runs is specified by `iterations`; in each run the initial
```

Comment:
```suggestion
The number of iterations is specified by `iterations`; in each *subsequent* iterations the initial
```

Reply:
I've changed this sentence in https://github.com/trixi-framework/Trixi.jl/commit/99539f980c07d3034eb2ddaf1b3f056e47821bae (although in a different way).